### PR TITLE
Implements feedback from @PixelantDesign

### PR DESCRIPTION
**Pull request by gdixon**
##### Description

This PR:

- [x] **Profile Page**-  When user navigates to the profile page from Grants Home, there was a note in the design spec to jump the user down to the Trust bonus section. Is this possible?

- [ ] ~**Profile Page** - Did not get a chance to review Idena as Graham noted it's not working on staging.~

- [x] **Profile Page** - When disconnecting from a service, there should be a confirmation at the top of the page. Please see designs.

- [x] **Profile Page**  - On ENS modal, please update the word validate in the last line to verify.

- [x] **Profile Page** - Disconnect Modals - The header should reflect that he user is disconnecting, the build currently says ' Connect to <Service Name>. Please see designs https://www.figma.com/file/CP57XgGQwC9ChPCMkTuQ7i/GR10?node-id=851%3A12660

- [x] **Save to Collection**  - After user clicks save to a collection, we redirect the user to see the collection they've just created. The Grant confirmation banner doesn’t display long enough for the user to read the confirmation message. It flashes very quickly, can we lengthen time to 3 seconds and then fade?

- [ ] ~**Saving Collections** - Also when saving to existing collection, is it possible to speed up the refresh not show the no data state collections while the grant page is refreshing?~

    _Could we open a separate ticket for this @PixelantDesign  - this sounds more like a bug than a regression?_

##### Refers/Fixes

Fixes: #9081

##### Testing

Tested locally

### CODE REVIEW

File: app/assets/v2/js/pages/profile-trust.js [hunk 1]
```diff
@@ -278,6 +278,7 @@ Vue.component('sms-verify-modal', {
       } finally {
         this.awaitingResponse = false;
         this.dismissVerification();
+        _alert("You have sucessfully disconnected your SMS verification.", "success", 3000);
```
Comment (zacheryschiller) [inline app/assets/v2/js/pages/profile-trust.js:281]: Typo `successfully`

File: app/assets/v2/js/pages/profile-trust.js [hunk 2]
```diff
@@ -494,6 +495,7 @@ Vue.component('twitter-verify-modal', {
       } finally {
         this.awaitingResponse = false;
         this.dismissVerification();
+        _alert("You have sucessfully disconnected from Twitter.", "success", 3000);
```
Comment (zacheryschiller) [inline app/assets/v2/js/pages/profile-trust.js:498]: Typo `successfully`

Comment (gdixon) [reply app/assets/v2/js/pages/profile-trust.js]: Resolved in c086cab

File: app/assets/v2/js/pages/profile-trust.js (hunk 1 above)
Comment (gdixon) [reply app/assets/v2/js/pages/profile-trust.js]: Resolved in c086cab

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**